misc/write_fake_manifests.py

```diff
@@ -76,7 +76,7 @@ def path(self):
 
     def src_obj_pairs(self, path, name):
         num_sources = paretoint(55, alpha=2) + 1
-        return [(os.path.join('..', '..', path, s + '.cc'),
+        return [(os.path.join('src', path, s + '.cc'),
```

Comment:
I made this change because I didn't want to write outside the target directory. What if I change the `-s, --sources` option from a boolean to taking a source directory argument?